billy-spain/src/main/java/com/premiumminds/billy/spain/SpainBootstrap.java

```diff
@@ -313,8 +313,8 @@ public Void runTransaction() throws Exception {
                                     Config.Key.Context.Spain.CanaryIslands.StaCruzDeTenerife.UUID);
 
                     // Taxes
-                    Date from = new DateTime(2019, 1, 1, 0, 0).toDateMidnight().toDateTime().toDate();
-                    Date to = new DateTime(2020, 1, 1, 0, 0).toDateMidnight().toDateTime().toDate();
+                    Date from = new DateTime(2020, 1, 1, 0, 0).toDateMidnight().toDateTime().toDate();
+                    Date to = new DateTime(2021, 1, 1, 0, 0).toDateMidnight().toDateTime().toDate();
```

Comment:
any reason to set an end date? pull request #69 fixed the need for this.

Reply:
The only reason I see is to force the anual review of this value, but that is thin. The date can be set to "infinity" if that seems better to you

Reply:
> The date can be set to "infinity" if that seems better to you
yes, please